apps/audits/api.py

```diff
@@ -58,6 +61,20 @@ class JobAuditViewSet(OrgReadonlyModelViewSet):
     ordering = ['-date_start']
 
 
+class JobsAuditViewSet(OrgModelViewSet):
+    model = Job
+    search_fields = ['creator__name']
+    filterset_fields = ['creator__name']
+    serializer_class = JobsAuditSerializer
+    ordering = ['-is_periodic', '-date_created']
+    http_method_names = ['get', 'options', 'patch']
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        queryset = queryset.exclude(type=Types.upload_file).filter(instant=False)
+        return queryset
+
+
 class FTPLogViewSet(OrgModelViewSet):
     model = FTPLog
     serializer_class = FTPLogSerializer
```

Comment:
The main differences found between the two models, `Job` and `JobsAudit`, seem to be:

  * The second model uses lowercase names for its fields, whereas the first one used uppercase.

There are no identified issues with this code. However, I noticed that we should avoid the repetition of `User Login Logs` when listing them in an API view, so we will update their serializers accordingly.
  
Here's a summary of our conclusions along with some suggestions:

Summary: We don't need any changes but there may be room for refactoring as suggested below.

Suggestions:
- Avoid repeating labels like UserLoginLog in the field lists (as noted above).
- Consider creating multiple serializers instead of serializing all three log types into one combined serializer (this can simplify serialization process).

Also consider using Django ORM attributes directly rather than accessing them through `.fields[]`. 

Let me know if you want to delve deeper on these points or have questions about specifics related to each change!